docs/notebooks/bsync_examples/Small-Office-GHG.ipynb

```diff
@@ -0,0 +1,1435 @@
+    "jan_ghg_elec += bsync.EndTimestamp(end_dt)\n",
+    "\n",
+    "# next, we convey the IntervalReading, i.e. the value.\n",
+    "# The value should use the same units as declared by the ResourceUse/ResourceUnits\n",
+    "jan_ghg_elec += bsync.IntervalReading(250.0)\n",
+    "\n",
+    "# finally, we need to convey to which resource this timeseries element is 'connected',\n",
+    "# i.e. the value represented by this reading conveys information for the Electricity resource use\n",
+    "jan_ghg_elec += bsync.ResourceUseID(IDref=elec_ru['ID'])\n",
+    "\n",
+    "# and we add this as a child to the TimeSeriesData element\n",
+    "ts_data += jan_ghg_elec\n",
+    "pretty_print(ts_data)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c3635e98",
+   "metadata": {},
+   "source": [
+    "#### All TimeSeries Data\n",
+    "The following cell simply performs the following:\n",
+    "1. Create a simple function to generate monthly timeseries elements for each resource use\n",
+    "1. Add it to a `TimeSeriesData` parent element\n",
+    "1. Add this back to the previously defined current building measured scenario (i.e. Scenario-1)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c8fb70b3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "full_ts_data = bsync.TimeSeriesData()\n",
+    "\n",
+    "# usage and peak data\n",
+    "monthly_elec = [6792.89, 5841.75, 6025.19, 4985.3, 5184.04, 5358.55, 5755.67, 5981.78, 5401.94, 5225.84, 5672.15, 6291.63]\n",
+    "monthly_ghg = [250.0, 240.0, 260.0, 250.0, 260.0, 230.0, 280.0, 270.0, 260.0, 250.0, 240.0, 250.0]\n",
+    "monthly_ng = [5.7, 4.01, 0.58, 0.4, 0.02, 0, 0, 0, 0, 0.01, 0.36, 6.08]\n",
+    "monthly_elec_peak = [15.42, 15.5, 16.25, 16.65, 18.56, 20.01, 20.82, 21.12, 20.42, 20.08, 17.4, 16.3]\n",
+    "\n",
+    "# costs data\n",
+    "monthly_elec_cost = [520.48, 451.53, 464.83, 389.43, 403.84, 416.49, 445.29, 461.68, 419.64, 406.87, 439.23, 484.14]\n",
+    "monthly_ng_cost = [30.44, 21.41, 3.10, 2.14, 0.11, 0.00, 0.00, 0.00, 0.00, 0.05, 1.92, 32.47]\n",
+    "\n",
+    "elec_ids = []\n",
+    "ng_ids = []\n",
+    "def create_monthly(values, resource_use_id, start_year, tsrq='Energy', rt='Total'):\n",
+    "    \"\"\"\n",
+    "    tsrq: One of Energy, Power, Cost\n",
+    "    rt: One of Total, Peak, Cost\n",
+    "    \"\"\"\n",
+    "    monthly = []\n",
+    "    end_year = start_year\n",
+    "    for i, val in enumerate(values, start=1):\n",
+    "        if i % 12 == 0:\n",
+    "            start_month = 12\n",
+    "            end_month = 1\n",
+    "            end_year += 1\n",
+    "        else:\n",
+    "            start_month = i % 12\n",
+    "            end_month = start_month + 1\n",
+    "        start_dt = datetime(start_year, start_month, 1)\n",
+    "        end_dt = datetime(end_year, end_month, 1)\n",
+    "\n",
+    "        my_id = f\"TS-{resource_use_id}-{tsrq}-{start_month}\"\n",
```

Comment:
```suggestion
    "        if tsrq == 'Greenhouse Gas Emissions':\n",
    "            my_id = f\"TS-{resource_use_id}-{start_month}\"\n",
    "        else:\n",
    "            my_id = f\"TS-{resource_use_id}-{tsrq}-{start_month}\"\n",
```
The `resource_use_id` already adds `GreenhouseGasEmissions` to the `ID`, and `tsrq` would introduce spaces into `ID` when it's related to ghg.